test/token/otcEscrow.spec.ts

```diff
@@ -261,7 +261,8 @@ describe("OtcEscrow", () => {
       }
 
       it("should revert", async () => {
-        await expect(subject()).to.be.revertedWith("unauthorized");
+        await subject();
```

Comment:
Put this `subject` in a beforeEach block, we want to make sure all the test set up doesn't happen inside the test block.